modules/openapi-generator/src/test/resources/3_0/petstore-with-fake-endpoints-models-for-testing.yaml

```diff
@@ -573,7 +571,6 @@ paths:
       parameters:
         - name: enum_header_string_array
           in: header
-          explode: true
```

Comment:
Seems like there's still an issue with the online converter. I'll open an issue tomorrow.